04-understanding-survey-data-documentation.Rmd

```diff
@@ -170,13 +159,13 @@ For weight | Use variance unit/PSU/cluster | and use variance stratum
 V200010a| V200010c| V200010d
 V200010b| V200010c| V200010d
 
-As mentioned above, we want to conduct ANES data analysis with weights to accurately represent the population. The user guide references a supplemental document called "How to Analyze ANES Survey Data"^[DeBell, Matthew. 2010. How to Analyze ANES Survey Data. ANES Technical Report Series no. nes012492. Palo Alto, CA, and Ann Arbor, MI: Stanford University and the University of Michigan] as a 'how-to guide' to help us with our analysis.
+As mentioned above, we want to conduct ANES data analysis with weights to accurately represent the population. The user guide references a supplemental document called "How to Analyze ANES Survey Data"^[DeBell, Matthew. 2010. How to Analyze ANES Survey Data. ANES Technical Report Series no. nes012492. Palo Alto, CA, and Ann Arbor, MI: Stanford University and the University of Michigan] as a 'how-to guide' to help us with our analysis. 
```

Comment:
There is a bit of a cognitive jump to get to this paragraph.  We should really walk people through step by step and provide as much information as we can about this.  We don't talk ANES weights summing to the sample and why we would want to sum to the population.  We should provide this detail.